src/inapp/utils.ts

```diff
@@ -591,27 +621,27 @@ export const paintIFrame = (
         };
 
         if (position !== 'Full') {
-          const iframeHeight =
-            iframe.contentWindow?.document?.body?.scrollHeight;
-          /*
-            For web in-app messages created with WYSIWYG editor,
-            there is 8px margin all around the iframe document body.
-            Add 16px to the iframe height to eliminate scrollbar.
-            Add 10px to eliminate discrepancy between the height
-            of the document html and that of the iframe itself.
-          */
-          iframe.style.height =
-            ((iframeHeight && iframeHeight + 26) || 0) + 'px';
```

Comment:
This is now handled by unsetting the margin set by the browser's default user agent stylesheet